### ISSUE TITLE
IE10 and IE9 Unable to get property 'store' of undefined or null reference

### ISSUE THREAD

**Issue by yoieh** [ryandrewjohnson/react-localize-redux#141]
**Do you want to request a _feature_ or report a _bug_?**
bug

**What is the current behavior?**
Crashes and Im getting an error in IE10 and IE9

```
SCRIPT5007: Unable to get property 'store' of undefined or null reference
bundle.js (91863,1)

The above error occurred in the <LocalizeProvider> component:
    in LocalizeProvider
    in PersistGate
    in Provider
```

```
 function LocalizeProvider(props) {
    _classCallCheck(this, LocalizeProvider);

    var _this = _possibleConstructorReturn(this, (LocalizeProvider.__proto__ || Object.getPrototypeOf(LocalizeProvider)).call(this, props));
   
/* This line below "bundle.js (91863,1)" */
==> var dispatch = _this.props.store ? _this.props.store.dispatch : _this.dispatch.bind(_this); 

    _this.getContextPropsSelector = Object(__WEBPACK_IMPORTED_MODULE_2__LocalizeContext__["b" /* getContextPropsFromState */])(dispatch);
```

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem. Your bug will get fixed much faster if we can run your code. Paste the link to your JSFiddle (https://jsfiddle.net/Luktwrdm/) or CodeSandbox (https://codesandbox.io/s/new) example below:**

I have tyed with both `LocalizeProvider`s `store` prop and with out 
```
ReactDOM.render(
  <Provider store={store}>
    <PersistGate loading={<LoadingScreenComponent show={true} />} persistor={persistor}>
      <LocalizeProvider>
        <ConnectedRouter history={history}>
          <App />
        </ConnectedRouter>
      </LocalizeProvider>
    </PersistGate>
  </Provider>,
  document.getElementById('root')
);
```

**Which versions of `react` and `react-localize-redux` are you using?**
    "react": "^16.6.3",
    "react-localize-redux": "^3.5.1",

**Comment by ryandrewjohnson** [comment]
@yoieh It looks like you need a couple of polyfills since you are supporting older browsers. I don't want to include these in the source, as this would increase the library size for all users. I will create a new issue to update the docs with how to support IE. 

I was able to get the demo app running in IE by adding a polyfill for `Symbol` and `Map`. If you aren't already I recommend using [Babel](https://babeljs.io/), which will make adding polyfills easier.

**Comment by yoieh** [comment]
Yes i do use babel 

``` .babelrc
{
    "presets": [
        "@babel/preset-env",
        "@babel/preset-react"
    ],
    "plugins": [
        "@babel/plugin-syntax-dynamic-import",
        "@babel/plugin-proposal-class-properties",
        "@babel/plugin-transform-proto-to-assign"
    ]
}
```
I have imported `@babel/polyfill` in index/entry file to deal orther problems of older browsers and should cover ES2015+ `Symbol` and `Map` if Im reading right. But yes a polyfill would probobly get this to work...

Any specific plugins your recomend to get `react-localize-redux` to work or do i need to wait for the docs to update?

**Comment by ryandrewjohnson** [comment]
Issue will be addressed in #142
